src/com/cloudogu/ces/cesbuildlib/HttpClient.groovy

```diff
@@ -51,23 +51,23 @@ class HttpClient implements Serializable {
     }
 
     protected String getCurlAuthParam() {
-        "-u ${script.env.CURL_USER}:${script.env.CURL_PASSWORD}"
+        "-u '${script.env.CURL_USER}:${script.env.CURL_PASSWORD}'"
```

Comment:
Check if this can be an security risk as the username can contain a single quote so that other evaluations like `${...}` can get injected. This can possibly be fixed by escaping quotes with a backslash or smth.